File: andy-server/server-user-center/src/main/java/com/andy/service/servierusercenter/dao/PmPageDao.java
package com.andy.service.servierusercenter.dao;


import com.andy.service.servierusercenter.entity.PmPageEntity;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.jpa.repository.JpaSpecificationExecutor;

import java.util.Optional;

/**
 * 作者：候帅
 * 时间：2018:09:01-19:26
 * 描述： 页面元素实体内容
 */

public interface PmPageDao extends JpaRepository<PmPageEntity, String>, JpaSpecificationExecutor<PmPageEntity> {

    Optional<PmPageEntity> findByPageCoding(String pageCode);
}

File: andy-server/servier-user-center/src/main/kotlin/com/andy/service/servierusercenter/dao/PmPageDao.kt
package com.andy.service.servierusercenter.dao;

import com.andy.service.servierusercenter.entity.PmPageEntity
import org.springframework.data.jpa.repository.JpaRepository

/**
 * FileName: UserDao
 * author:   候帅
 * data:     23/08/2018 22:04
 * Description: TODO
 * History:
 */
interface PmPageDao: JpaRepository<PmPageEntity, Long> {
}